nixos/modules/services/networking/shairport-sync.nix

```
@@ -100,12 +102,12 @@ in
         serviceConfig = {
           User = cfg.user;
           Group = cfg.group;
-          ExecStart = "${pkgs.shairport-sync}/bin/shairport-sync ${cfg.arguments}";
+          ExecStart = "${cfg.package}/bin/shairport-sync ${cfg.arguments}";
```

Using lib.getExe is preferred to using /bin
```suggestion
          ExecStart = "${lib.getExe cfg.package} ${cfg.arguments}";
```

---

Great to know, thank you! Updated.